kubernetes/README.md

```diff
@@ -4,70 +4,111 @@ This example explains how to deploy the 1Passwrd SCIM bridge on Kubernetes runni
+As mentioned before, you'll need to configure you DNS after the fact, as configuring DNS requires the IP address of the deployed container.
 
-NOTE: If you are using an existing redis instance that is not running on `redis:6379`, add the `--redis-host=[host]` and `--redis-port=[port]` flags to `containers.args` in the deployment yaml file.
+It's important that you use the IP address of the container that runs the load balancer. The load balancer is what handles traffic for ports 80 and 443 for the SCIM bridge. Here are the mappings:
 
-Example:
 ```
-kubectl apply -f op-scim-deployment.yaml
-kubectl apply -f op-scim-service.yaml
+Load Balancer: 80 -> SCIM Bridge: 8080
+Load Balancer: 443 -> SCIM Bridge: 8443
 ```
 
-These files configure the 1Password SCIM Bridge to connect to the redis instance indicated by the args, and it deploys a GCP load balancer to handle traffic on ports 80 and 443. Traffic on :80 is needed to perform the LetsEncrypt certificate challenges, after which all SCIM traffic will be served on :443
+Within your DNS provider, create the DNS record for the domain you decided on in _Start creating your DNS record_. You should give it a few minutes to propagate. This can take anywhere from 5 minutes to an hour depending on what DNS provider you're using for the domain.
+
+You may or may not need to reboot the containers you've deployed to ensure the LetsEncrypt challege service finishes correctly, but you should see if it automatically completes on its own before doing so.
 
-NOTE: Port 80 on the load balancer is forwarded to :8080 on the SCIM Bridge, and port 443 is forwarded to :8443.
 
-At this point you should create your DNS record using the external IP address of the load balancer and wait for it to propogate.
+## Test the instance
 
-Once the record is propogated, you can test your instance by requesting `https://[your-domain]/scim/Users`, with the header `Authorization: Bearer [bearer token]` which should return a list of the users in your 1Password account.  You can do this with `curl` as follows:
+Once the DNS record has propagated, you can test your instance by requesting `https://[your-domain]/scim/Users`, with the header `Authorization: Bearer [bearer token]` which should return a list of the users in your 1Password account. 
 
+You can do this with `curl`, as an example:
 ```
 curl --header "Authorization: Bearer <bearertoken>" https://<domain>/scim/Users
 ```
 
-Alternatively, visit the domain you configured earlier.  You'll see a 1Password SCIM Bridge Status page which can be used to verify your OAuth bearer token.
+Alternatively, you can visit the domain via any web browser. You'll see a 1Password SCIM Bridge Status page which can be used to verify your OAuth bearer token. It is safe to put the bearer token in this page.
```

Comment:
> It is safe to put the bearer token in this page.

Why would it not be? Seeing this, should I be suspicious of other deployments? Are there fake/malicious scim bridges out there?

Instead I would say something like:
- This page is being served by your SCIM Bridge using a secured TLS connection established using your LetsEncrypt domain certificate.